### PR TITLE
OV-147: Create create avatar page

### PR DESCRIPTION
**Pull request by stefano-lacorazza**
Created create avatar page
image

Branch created from task/OV-164

### CODE REVIEW

File: frontend/src/bundles/create-avatar/components/instruction/instruction.module.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    border-radius: 50%;
+    background-color: white;
+    color: var(--chakra-colors-brand-secondary-50);
+}
+
+#iconAvoid {
+    border-radius: 50%;
+    background-color: white;
+    color: var(--chakra-colors-brand-secondary-900);
+}
```
Comment (o-nedashkivska) [inline frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:34]: Why do we need ids here? Class styles get overridden?

File: frontend/src/bundles/create-avatar/pages/create-avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const CreateAvatar: React.FC = () => {
+    return (
+        <Box bg="background.50" height="calc(100vh - 75px)" p="50px">
+            <Box
+                sx={{
+                    maxWidth: '870px',
+                    marginLeft: 'auto',
+                    marginRight: 'auto',
+                }}
```
Comment (o-nedashkivska) [inline frontend/src/bundles/create-avatar/pages/create-avatar.tsx:13]: Let's move it to css module

File: frontend/src/bundles/create-avatar/components/instruction/instruction.module.css (hunk 1 above)
Comment (stefano-lacorazza) [reply frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:34]: In icons chakra theme colors are not showing if we I className but they do if I use id.

Comment (nikita-remeslov) [reply frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:34]: pull next branch with fix and try to use className

File: frontend/src/bundles/create-avatar/components/instruction/instruction.module.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+.textSecondary50 {
+    color: var(--chakra-colors-brand-secondary-50);
+}
+
+.textSecondary900 {
+    color: var(--chakra-colors-brand-secondary-900);
+}
```
Comment (nikita-remeslov) [inline frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:42]: looks like you dont use these classes

File: frontend/src/bundles/create-avatar/components/instruction/instruction.module.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+.unorderedList {
```
Comment (nikita-remeslov) [inline frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:44]: you can use default prop to style it

File: frontend/src/bundles/create-avatar/components/instruction/instruction.module.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    max-width: 300px;
+}
+
+.button {
```
Comment (nikita-remeslov) [inline frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:48]: you can use default prop to style it

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/create-avatar/components/instruction/instruction.module.css (hunk 4 above)
Comment (stefano-lacorazza) [reply frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:44]: At the moment Button only accepts props: (label, type, variant, isDisables, ClassName, onCLick) so I cant pass maxW as a prop, should I modify the button so it accepts all chakra stylings as direct props?

Comment (stefano-lacorazza) [reply frontend/src/bundles/create-avatar/components/instruction/instruction.module.css:36]: I did it in https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/178 which is this branch's parent.

File: frontend/src/bundles/create-avatar/components/instruction/instruction.tsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                        <ListItem>
+                            Keep your head centered in the frame
+                        </ListItem>
+                        <ListItem>
+                            Ensure face visibility; do not cover your mouth
+                        </ListItem>
+                        <ListItem>
+                            Use a tripod or stabilize your phone to avoid
+                            shaking
+                        </ListItem>
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/create-avatar/components/instruction/instruction.tsx:46]: If its a list maybe we can move it to array and map it?

File: frontend/src/bundles/create-avatar/components/instruction/instruction.tsx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+                        <Text
+                            as="span"
+                            color="brand.secondary.900"
+                            variant="bodySmall"
+                        >
+                            Things to avoid
+                        </Text>
+                    </Flex>
+                    <UnorderedList max-width= "300px">
+                        <ListItem>Stitches or cuts of your footage</ListItem>
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/create-avatar/components/instruction/instruction.tsx:65]: same here

File: frontend/src/bundles/create-avatar/pages/create-avatar.module.css [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+    background-color: var(--chakra-colors-background-50);
+    height: calc(100vh - 75px);
+    padding: 50px;
+}
+
+.innerContainer {
+    max-width: 870px;
+    margin-left: auto;
+    margin-right: auto;
+}
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/create-avatar/pages/create-avatar.module.css:11]:
```suggestion
}

```

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

**Review by o-nedashkivska — APPROVED** [review]

**Comment by stefano-lacorazza** [comment]
I had some problems with the last merge of next and there are more files in "Files Changed" that they should be. I dont know how to change it.

**Review by anton-otroshchenko — DISMISSED** [review]

**Review by sergiy4 — DISMISSED** [review]

**Review by sofiia-trokhymchuk — CHANGES_REQUESTED** [review]
Use kebab-case for CSS class names

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**